tests/unit_tests/Request/RequestTest.php

```diff
@@ -228,31 +227,31 @@ public function testRequestRetryFails()
         $request_options = [$method, $uri, $this->request_headers, null,];
 
         $this->container->expects($this->at(0))
-          ->method('get')
-          ->with(Client::class, [$this->client_options,])
-          ->willReturn($this->client);
+            ->method('get')
+            ->with(HttpRequest::class, $request_options)
```

Comment:
The order in which these are being drawn from the container is reversed, but it doesn't affect the result